elliottlib/cli/common.py

```diff
@@ -64,6 +64,11 @@ def print_version(ctx, param, value):
     help='Show debug output on console.')
 @click.option("--brew-event", metavar='EVENT', type=click.INT, default=None,
               help="Lock koji clients from runtime to this brew event.")
+@click.option("--bug-mode", metavar="BUG_MODE", default='both',
```

Comment:
This is the major change in this PR. With this we'll be able to explicitly specify 
which mode we want to use - cleaner than env vars